docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1127,9 +1138,44 @@ function clearMediaDeviceList() {
   sourceDevicesVideoInput.innerText = '';
 }
 
+function getLocalMediaSettings() {
+  const meeting = getCurrentMeeting();
```

Comment:
We will need to change this to get the setting every 3 second at-least to get the realtime data ,

local and remote resolution will change based on network condition and other parameter so this need to be captured every 3 second ,  can start the interval on media:ready

Reply:
Done